packages/scene-composer/src/store/slices/SceneDocumentSlice.ts

```diff
@@ -202,7 +202,7 @@ export const createSceneDocumentSlice: SliceCreator<keyof ISceneDocumentSlice> =
               ]);
             });
         } else {
-          appendSceneNode(draft as RootState, node, disableAutoSelect);
```

Comment:
Should I not have made this change? Is this cast important?

Reply:
If it still loads saved static and dynamic scenes then this cast removal should be fine.